<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QListWidget" name="user_list">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>121</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="shop_list">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>121</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="rider_list">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>120</y>
     <width>121</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="all_te">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>120</y>
     <width>361</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="chat_le">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>570</y>
     <width>391</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="chat_send">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>570</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>보내기</string>
   </property>
  </widget>
  <widget class="QListWidget" name="chat_list">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>360</y>
     <width>201</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="user_lb">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>유저 0명</string>
   </property>
  </widget>
  <widget class="QLabel" name="shop_lb">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>320</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>매장 0명</string>
   </property>
  </widget>
  <widget class="QLabel" name="rider_lb">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>320</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>라이더 0명</string>
   </property>
  </widget>
  <widget class="QLabel" name="all_lb">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>320</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>총 0명</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="chat_te">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>360</y>
     <width>521</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
